crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -455,7 +459,7 @@ impl<T: WasmInstance> WasmModuleInstance<T> {
         let ctx = ExecutionContext::reducer(address, ReducerContext::from(op.clone()));
```

Comment:
This still hasn't been removed.

Reply:
We can create a follow up ticket for this one.

Reply:
Context: Is not possible to remove it without causing a deadlock, so is better to wait for a proper investigation.